src/components/SigninButtons/index.js

```diff
@@ -29,10 +29,13 @@ const Buttons = ({
     </div>
 
     <div className="row">
-      <Link className="input-field col s12 m6 link--icon" to="/reset/password">
+      <Link className="input-field col s12 m6 link--icon" to="/login">
         <svg className="icon icon--default">
           <use xlinkHref="/ui/static/assets/icons/sprite.svg#icon-arrow-left" />
         </svg>
+             Back
+      </Link>
+      <Link className="input-field col s12 m6 link--icon" to="/reset/password">
```

Comment:
One thing, I would like to help us with in your PR is creating a `ROUTES` file which will contain an object with all **urls** and we can access our routes for instance with: 
- `ROUTES.home` for `/` 
- `ROUTES.login` for `/login`.

This will help us since we will have one central point to control our Routes, makes sense? 🤔

Reply:
You mean have a component with `Link` components?